dev-docs/source/auth.rst

```diff
@@ -80,3 +80,47 @@ See the :doc:`wfs` documentation for more details.
 
     When changing the authorization logic, make sure to test the WFS server endpoint too.
     While most logic is shared, it's important to double-check no additional data is exposed.
+
+
+Testing
+-------
+
+When testing datasets with authorization from the command line,
+you can generate a JWT with the following script:
+::
+
+    import json
+    import sys
+    import time
+    
+    from dso_api.settings import JWKS_TEST_KEY
```

Comment:
In Django doe je altijd:

```
from django.conf import settings

settings.JWKS_TEST_KEY
```

Dan kan je namelijk met de `$DJANGO_SETTINGS_MODULE` of `manage.py --settings=...` de settings module opgeven.